qiskit/algorithms/__init__.py

```diff
@@ -110,6 +110,8 @@
     TrotterQRTE
     EvolutionResult
     EvolutionProblem
+    ScipyRealEvolver
+    ScipyImaginaryEvolver
```

Comment:
Not sure how it relates to naming conventions in Python, but the correct spelling of the package is SciPy.